VERSION

```diff
@@ -1 +1 @@
-1.3.4
\ No newline at end of file
+1.4.4
```

Comment:
Bumping the Minor version should reset the patch version to 0.

Should be just 1.4.0